2fa-api/handlers/create.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../db");
+
+const create = function (req, res) {
+  try {
+    const { email } = req.body;
+    const { password } = req.body;
+    const { phone } = req.body;
+
+    db.connection.query(
+      `INSERT INTO Users (email, password, phone) VALUES ('${email}', '${password}', '${phone}')`,
+      (error, results) => {
+        if (error) {
+          console.log(error);
+          return res.status(500).json({ error });
+        }
+
+        req.session.loggedIn = true;
+        return res.status(200).json({ data: { loggedIn: true, results } });
```

Comment:
can you think about what would be wrong with returning the entire `results` object?

Reply:
Overfetching, also returning the password in its original form.

Reply:
+ the `id` of the record. why does the user need _any_ of this? major security flaw

Reply:
fixed